sigs.yaml

```diff
@@ -0,0 +1,166 @@
+# Copied from github.com/kubernetes/community - used for devstats sql generation
+sigs:
+- dir: sig-compute
+  name: compute
+  subprojects:
```

Comment:
Would it make sense to have a subproject for each separate repo? If there are different OWNER files in each repo, then include that in the `owners` section, wdyt? 

Anyway, AIUI the end result https://gist.github.com/dhiller/88ba550b04c8006418b8268c5bb73b8c would be the same, is that correct?

Reply:
Yeah, I think that could make sense. This was just a first shot, so I think there's definitely room for improval here (aka automation of scanning for all OWNERS files per repo and then generating the subprojects from that)

Reply:
I was just pinning this down to get a feel of what you folks think about the idea, I think that the categories aren't quite right ATM